integreat_cms/cms/templates/_form_language_tabs.html

```diff
@@ -68,49 +73,15 @@
-                        {% for other_language, other_status in sorted_translation_states|slice:"4:" %}
-                            <a href="{% build_url target=target region_slug=request.region.slug language_slug=other_language.slug content_field=content_field content_id=instance.id %}"
-                               class="flex justify-between font-bold whitespace-nowrap {% if forloop.last %}rounded-b border-b{% endif %} py-2 px-3 border-l border-r border-t border-blue-500 hover:bg-water-500 bg-white">
-                                <div>
-                                    {% if other_status == translation_status.IN_TRANSLATION %}
-                                        <span title="{% translate "Currently in translation" %}">
-                                            <i icon-name="clock"></i>
-                                        </span>
-                                    {% elif other_status == translation_status.OUTDATED %}
-                                        <span title="{% translate "Translation outdated" %}">
-                                            <i icon-name="alert-triangle"></i>
-                                        </span>
-                                    {% elif other_status == translation_status.UP_TO_DATE %}
-                                        <span title="{% translate "Translation up-to-date" %}">
-                                            <i icon-name="check"></i>
-                                        </span>
-                                    {% elif other_status == translation_status.MISSING %}
-                                        <span title="{% translate "Translation missing" %}" class="no-trans">
-                                            <i icon-name="x"></i>
-                                        </span>
-                                    {% elif other_status == translation_status.MACHINE_TRANSLATED %}
-                                        <span title="{% translate "Machine translated" %}">
-                                            <i icon-name="bot"></i>
-                                        </span>
-                                    {% endif %}
-                                    <span class="px-2 max-w-max truncate">{{ other_language.translated_name }}</span>
-                                </div>
-                                <div>
-                                    <span class="fp fp-rounded fp-{{ other_language.primary_country_code }} ml-2"></span>
-                                    {% if other_language.secondary_country_code %}
-                                        <span class="fp fp-rounded fp-{{ other_language.secondary_country_code }}"></span>
-                                    {% endif %}
-                                </div>
-                            </a>
-                        {% endfor %}
+                    <div id="language-switcher-list" class="z-10 hidden h-0">
```

Comment:
It would be better to change it to `ul` tag because it contains list items :thinking: